ui/new-private-messages.js

```diff
@@ -20,25 +20,43 @@ module.exports = function (state) {
           this.$route.matched[0].instances.default.renderPrivate()
         else
           this.$router.push({ path: '/private'})
-      }
-    },
-
-    created: function () {
-      var self = this
+      },
 
-      if (loaded) return // is loaded twice?
-      loaded = true
+      tryLoading: function () {
+        var self = this
+  
+        if (loaded) return // is loaded twice?
+        loaded = true
+  
+        [ err, SSB ] = ssbSingleton.getSSB()
+        if (err) {
+          setTimeout(self.tryLoading, 3000)
+          return
+        }
+  
+        // This should only be done once, and only after we've already gotten an SSB running, which is why it's done here.
+        if (!self.registeredSSBChange) {
```

Comment:
Isn't there already a loaded check that will make sure we are not called twice? Or why is this needed?

Reply:
Both of these are there because they were there before and I didn't bother to track down why.  :)

I'll check and see if they can be eliminated.

Reply:
Great! Thanks, it just looking wierd when I was reviewing. The other stuff was quite straightforward.

Reply:
Just eliminated it, because I don't see any reason why it's actually needed.  Also had to add semicolons to a couple of the lines because if you do a variable assignment right before doing an array assignment without a semicolon to terminate it, the whitespace is flattened and bad things happen.